src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2115,42 +2219,60 @@
           short-summary: Port number on which to receive traffic.
         - name: --dest-address
           short-summary: The IP address or URI at which to receive traffic.
+    examples:
+        - name: Checks connectivity between two virtual machines in the same resource group over port 80.
```

Comment:
Remove plural, change to `Check`. Applies to following description as well.

Reply:
Fixed in my branch